libs/internal/src/data_model/flag.cpp

```diff
@@ -26,4 +26,26 @@ Flag::Rollout::Rollout(std::vector<WeightedVariation> variations_)
       bucketBy("key"),
       contextKind("user") {}
 
+bool Flag::IsExperimentationEnabled(
```

Comment:
Just moving this from where the `AllFlagsState` needed it internally into a public place, since it's used more broadly now.